models/activity.ts

```diff
@@ -0,0 +1,55 @@
+import { list } from "@keystone-6/core";
+import type { ListConfig } from "@keystone-6/core";
+import type { Lists } from ".keystone/types";
+import {
+  integer,
+  text,
+  timestamp,
+  relationship,
+} from "@keystone-6/core/fields";
+
+export const Activity: ListConfig<Lists.Activity.TypeInfo<any>, any> = list({
+  access: {
+    operation: {
+      query: ({ session }) => !!session,
+      create: ({ session }) => !!session,
+      update: ({ session }) => !!session,
+      delete: ({ session }) => !!session,
+    },
+    item: {
+      update: ({ session, item }) => item.userId === session.data.id,
+      delete: ({ session, item }) => item.userId === session.data.id,
+    },
+  },
+  fields: {
+    name: text({
+      defaultValue: "...",
```

Comment:
This defaultValue looks weird in Keystone when it loads (see screenshot). I recommend sticking to figma on this and leaving it blank. Then when we build the form on the frontend we can input those placeholder values shown in figma. 

![image](https://github.com/user-attachments/assets/9b4fe2b6-c6d6-4673-b32b-4c78eaebdd96)

Reply:
I made the default values blank now.